src/main.ts

```diff
@@ -55,12 +65,19 @@ ws.onclose = () => {
 
 ws.onmessage = (e) => {
   const data = JSON.parse(e.data);
-  if (data.id == 'notifications') {
-    console.log('received data', data);
-    if (data.payload?.errors) {
-      console.log('Errors', data.payload.errors);
-    } else if (data.payload?.data?.charleslavon_near_n0_notifications) {
-      console.log(data.payload?.data?.charleslavon_near_n0_notifications);
+  if (data.payload?.errors) {
+    console.log('Errors', data.payload.errors);
+  } else if (data.id == 'notifications' && data.payload?.data?.charleslavon_near_n0_notifications) {
+    const messages = data.payload?.data?.charleslavon_near_n0_notifications;
+    console.log('Received data', messages);
+
+    if (messages.length > 0) {
+      messages.forEach( (message) => {
+        console.log('Publishing message', message);
+        topic.publishMessage({data: Buffer.from(JSON.stringify(message))});
+        console.log('Message published');
+
+      });
```

Comment:
I like early returns on blocks like this as opposed to else-if, but either is fine

e.g.
```ts
  if (data.payload?.errors) {
    console.log('Errors', data.payload.errors);
    return;
  }

  if (data.id !== 'notifications' || !data.payload?.data?.charleslavon_near_n0_notifications) {
    console.log('Not configured to handle payload:', payload);
    // likely only do the above log if in debug/local mode
    return;
  }

  const messages = data.payload?.data?.charleslavon_near_n0_notifications;
  console.log('Received data', messages);
  // etc...
```